app/controllers/providers/no_eligibility_assessments_controller.rb

```diff
@@ -1,12 +1,27 @@
 module Providers
   class NoEligibilityAssessmentsController < ProviderBaseController
     def show
+      submission
+      write_cfe_result
       legal_aid_application.provider_enter_merits! unless legal_aid_application.provider_entering_merits?
```

Comment:
I don't think you need to call `#submission` here -you can just refer to `submission.id` instead of `@submission.id` in the `#write_cfe_result` method and it will do it there.

Reply:
change made